packages/frontend/src/utils/stages.js

```diff
@@ -1,7 +1,7 @@
 const GAME_STAGES = Object.freeze({
-  SETUP: 'SETUP',
-  PLAYING: 'PLAYING',
-  GAME_OVER: 'GAME_OVER',
+  SETUP: 0,
+  PLAYING: 1,
+  GAME_OVER: 2,
```

Comment:
Since they correspond to our route structure and routes is an array of objects, I'm using indexes.

Reply:
That makes sense, I'm just not seeing when it'd be used where it would make a difference.

Reply:
If it's the `useEffect` in `App.jsx` which changes the navigation path, switching that logic to be based on the path that's entered in the browser would probably make it so we wouldn't need the indices of the routes. I don't think the indices of the routes will be needed other than when rendering the routes. If we need reference to them otherwise, we could always use `Array.find`

Reply:
I really think these would make more sense as strings instead of numbers

Reply:
@perennialAutodidact  I'm not understanding. The purpose of that use effect is to set the app back to the correct route if the user enters an invalid url. It needs the path based off the stage of the game. How would we do this without using the game stage to reference the route?